rapidtide/correlate.py

```diff
@@ -18,36 +18,27 @@
 # $Author: frederic $
 # $Date: 2016/07/12 13:50:29 $
 # $Id: tide_funcs.py,v 1.4 2016/07/12 13:50:29 frederic Exp $
-#
-from __future__ import print_function, division
+"""Functions for calculating correlations and similar metrics between arrays."""
+import sys
 
+import matplotlib.pyplot as plt
 import numpy as np
 import scipy as sp
-from scipy import fftpack, signal
-from numpy.fft import rfftn, irfftn
-import matplotlib.pyplot as plt
-
-import sys
+from numpy.fft import irfftn, rfftn
+from scipy import signal
 from sklearn.metrics import mutual_info_score
 
-import rapidtide.util as tide_util
-import rapidtide.resample as tide_resample
 import rapidtide.fit as tide_fit
 import rapidtide.miscmath as tide_math
+import rapidtide.resample as tide_resample
+import rapidtide.util as tide_util
```

Comment:
I ran `isort` here, which automatically sorts imports based on three groups: builtins, third-party (e.g., numpy), and within-package. Within each of those groups, it sorts `import X`s alphabetically, then `from X import Y`s alphabetically.

Reply:
Sounds like a good strategy going forward.